onnx/common/ir.h

```diff
@@ -992,6 +982,14 @@ friend struct Value;
     return opset_versions_;
   }
 
+  size_t getNextUnique() {
+      std::string next_unique_name = ONNX_NAMESPACE::to_string(++next_unique_);
+      while(!isNameUnique(next_unique_name)) {
+          next_unique_name = ONNX_NAMESPACE::to_string(++next_unique_);
+      }
+      return next_unique_;
+  }
+
```

Comment:
Is `getNextUnique` a const function? Did you move it to public because optimizer wants to use it?

Reply:
> Is getNextUnique a const function?

It is not const (but idempotent)

> Did you move it to public because optimizer wants to use it?

Yes. Since the methods like `addInitializer(Tensor initializer, std::string name)` is public, `getNextUnique` should also be public to get a unique name as the argument.